ui/lib/utils/reportData.ts

```diff
@@ -0,0 +1,17 @@
+import mixpanel from 'mixpanel-browser'
+import client from '@lib/client'
+
+export async function init() {
+  mixpanel.init('cb1135f29a413c653332990f07b3586a', {
```

Comment:
I opened a new project called `Dev` so that we can separate Dev data and production data. Then you can supply different token in different env by using `.env` file: https://create-react-app.dev/docs/adding-custom-environment-variables/

Reply:
Nice!